### PR TITLE
typo fixes for issues 72, 91, 95-98, 101, 122, 145

### PR DESCRIPTION
**Pull request by stewardont**
These were all simple typo fixes in literals. No URIs were changed.

Do I need to do something with the serializer before merging? I don't understand how the serializer fits into the workflow.

### CODE REVIEW

**Comment by sa-bpelakh** [comment]
@stewardont Steve, is there a reason you put the new versions into a separate directory, which means there are no meaning diffs in the PR? Also, what about the creation of the `gist` submodule - I have not worked with submodules previously in git.

**Comment by stewardont** [comment]
@sa-bpelakh the reason is general noob incompetence. I'll fix that and comment on the PR when it's worth continuing review.

**Comment by rjyounes** [comment]
@sa-bpelakh Just to fill you in on the process we agreed on today: Steve will assign reviewers to review his implementation. The assignees are those with permission to merge the PR to develop. I'll copy from our notes:

```
Once the reviewer(s) have approved the PR:

If the reviewer also has the ability to merge, he/she merges to develop

If the reviewer does not have the ability to merge, add a comment to the effect that the PR has been approved, calling out to one of the mergers (Rebecca, Boris, and MarkW) to do the merge to develop.
```
So you don't need to do anything with this issue until the reviewers have approved it (although are welcome to review also if you wish).

**Comment by rjyounes** [comment]
@stewardont will also run the serializer so that the files can meaningfully be diffed.

**Comment by uscholdm** [comment]
To SteveS: when you run the serializer, re-issued a review request so I can have a look.

**Comment by marksem** [comment]
@stewardont  and @uscholdm , if Steve juts re-pushes branch after serialization, it should notify reviewers by email that someone pushed new changes.

**Comment by uscholdm** [comment]
MarkW said:

>  @stewardont put fixes in a new folder/directory called typo fixes instead of just being done in the main 'OntologyFiles' folder. Looks like Steve did them directly with a text editor, so no serialization step would be needed in this case.

Oh, so putting them in a new folder resulted in the file-compare being useless, rather than it being a serialization issue.

**Comment by uscholdm** [comment]
@marksem said:
>   copy files frrom typo fixes folder to OntologyFiles folder
    **delete OntologyFiles folder**
    delete empty gist subfolder he accidentally created
    git push to update the PR

Huh??  Do you mean delete the typo fixes folder?

**Comment by rjyounes** [comment]
I assume we are still waiting on Steve's changes before reviewing. @marksem I think you said at the meeting that we would be informed when it's ready for review, but how does that work? The new edits go into the original pull request. How do we know when he's just working on the files vs finishing the requested changes to the PR?

**Comment by stewardont** [comment]
@rjyounes I did not change "Phenomena Type" to "Phenomena type" since we still need to revisit our case conventions.
I believe I fixed the directory structure with my earlier commit.
@rjyounes @uscholdm please review.

**Comment by rjyounes** [comment]
@stewardont Agreed, the comment about PhenomenaType was a mistake on my part based on misreading the diffs.

File: OntologyFiles/gistEvent.owl [hunk 1]
```diff
@@ -203,7 +203,7 @@
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&gist;hasGoal">
-		<rdfs:label rdf:datatype="&xs;string">Hs Goal</rdfs:label>
+		<rdfs:label rdf:datatype="&xs;string">Ha	 s Goal</rdfs:label>
```
Comment (rjyounes) [inline OntologyFiles/gistEvent.owl]: This needs a fix.

File: OntologyFiles/gistIoT.owl [hunk 2]
```diff
@@ -67,13 +67,13 @@
 	<owl:Class rdf:about="&gist;PhenomenaType">
 		<rdfs:subClassOf rdf:resource="&gist;Category"/>
 		<rdfs:label rdf:datatype="&xs;string">Phenomena Type</rdfs:label>
-		<rdfs:comment rdf:datatype="&xs;string">The things that a sensor can sense, such as light, heat, current, moisture et</rdfs:comment>
+		<rdfs:comment rdf:datatype="&xs;string">The things that a sensor can sense, such as light, heat, current, moisture etc.</rdfs:comment>
```
Comment (rjyounes) [inline OntologyFiles/gistIoT.owl]: Add comma after moisture (comma before "etc." is the most commonly recommended style).

**Comment by stewardont** [comment]
@rjyounes I deleted the blankspace you noticed and added that comma. Thank you.

File: OntologyFiles/gistIoT.owl [hunk 3]
```diff
@@ -67,13 +67,13 @@
...
-		<rdfs:comment rdf:datatype="&xs;string">The things that a sensor can sense, such as light, heat, current, moisture et</rdfs:comment>
+		<rdfs:comment rdf:datatype="&xs;string">The things that a sensor can sense, such as light, heat, current, moisture, etc.</rdfs:comment>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&gist;PhysicalActionType">
 		<rdfs:subClassOf rdf:resource="&gist;Category"/>
-		<rdfs:label rdf:datatype="&xs;string">Phenomena Type</rdfs:label>
-		<rdfs:comment rdf:datatype="&xs;string">The effects to be realized in the real world, such as lifting a garage door, turning off a valve, dropping cadmium rods etc</rdfs:comment>
+		<rdfs:label rdf:datatype="&xs;string">Physical Action Type</rdfs:label>
+		<rdfs:comment rdf:datatype="&xs;string">The effects to be realized in the real world, such as lifting a garage door, turning off a valve, dropping cadmium rods etc.</rdfs:comment>
```
Comment (rjyounes) [inline OntologyFiles/gistIoT.owl]: Sorry to be such a stickler - I see another comma before "etc." is needed here. After that I will approve it.

Comment (stewardont) [reply OntologyFiles/gistIoT.owl]: fixed

**Comment by rjyounes** [comment]
I'm merging the PR without Michael's review because the changes are quite trivial and have already gone through a couple of corrections.

**Pull request merged**